uropa/visualization.py

```diff
@@ -108,30 +123,71 @@ def peak_count_plot(table, var, kind):
     pass
 
 
-def upset_plot(table, var):
+def upset_plot(table, var="feature", peak_columns=["peak_chr", "peak_start", "peak_end", "peak_strand"], title=None, title_size=20, path=None, dpi=300.0):

```

Comment:
Please add the possibility for additional upsetplot parameters. See kwargs

Reply:
Has been added as requested.